db/migrate/20220825134619_add_uuid_to_exercise_representations.rb

```diff
@@ -0,0 +1,16 @@
+class AddUuidToExerciseRepresentations < ActiveRecord::Migration[7.0]
+  def change
+    add_column :exercise_representations, :uuid, :string, null: true, index: { unique: true }
+
+    # TODO: consider if we can run this in production
+    ActiveRecord::Base.transaction(isolation: Exercism::READ_COMMITTED) do
+      Exercise::Representation.find_each do |representation|
+        Exercise::Representation
+          .where(id: representation.id)
+          .update_all(uuid: SecureRandom.compact_uuid)
```

Comment:
This is not what you want :) This will set the uuid of all records to the same thing. Instead you want to use the Mysql UUID feature:
```suggestion
          .update_all("`uuid` = UUID()")
```

or if you really care about company uuids, you'd have to iterate every row (which means I think we don't care 😉)

Reply:
Doh

Reply:
> This will set the uuid of all records to the same thing.

Are you saying that `SecureRandom.compact_uuid` returns the same thing when called in a tight loop? Otherwise, I think I'm just updating one representation.

```ruby
Exercise::Representation
          .where(id: representation.id)
          .update_all(uuid: SecureRandom.compact_uuid)
```